samples/bluetooth/direction_finding_central/sysbuild/ipc_radio/prj.conf

```diff
@@ -2,20 +2,14 @@ CONFIG_IPC_SERVICE=y
 CONFIG_MBOX=y
 
 CONFIG_HEAP_MEM_POOL_SIZE=8192
-
-CONFIG_MAIN_STACK_SIZE=512
-CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=512
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
```

Comment:
Is the increase in memory usage due to the use of ipc_radio?

Reply:
The increase is precautionary. When tests manual and automatic improve the stack sizes can be limited to numbers similar to hci_ipc/multiprotcol_rpmsg numbers based on testing.